config/routes.rb

```diff
@@ -11,7 +11,6 @@
   get "/:sector/:subcategory/latest", as: "latest_changes", to: "subcategories#latest_changes"
   get "/:sector/:subcategory", as: "subcategory", to: "subcategories#show"
   get "/:sector", to: "specialist_sectors#show"
-
-  resources :email_signups, path: "/:subtopic/email-signups", only: [:create], subtopic: %r{[^/]+/[^/]+}
-  get "/:subtopic/email-signup", as: "email_signup", to: "email_signups#new", subtopic: %r{[^/]+/[^/]+}
+  get "/:sector/:subcategory/email-signup", to: "email_signups#new", as: "email_signup"
+  post "/:sector/:subcategory/email-signup", to: "email_signups#create"
```

Comment:
Note: previously the GET route ended `/email-signups` (plural) whereas the POST route ended `/email-signup`.  I'm not sure there's a good reason for that, but if we're going to change it, I think that should be in a separate PR (and include a redirect from the old location...)

Another point.  While we're renaming the params, it's probably worth changing to use the correct naming (ie topic/subtopic).  They could be `topic_slug` and `subtopic_slug` to make it very obvious what's expected in them.